src/components/biblio/BiblioFileManagement.js

```diff
@@ -338,7 +384,6 @@ const FileEditor = () => {
                             onChange={(event) => patchReferencefile(referenceFile.referencefile_id, {"pdf_type": event.target.value === "" ? null : event.target.value}, accessToken)}>
                 <option></option>
                 <option>ocr</option>
```

Comment:
@sweng66 This tif option shouldn't be removed.  It probably happened from some merge oversight ?

Reply:
Great catch!! Thanks @azurebrd 

Added back